src/core/expr/py_join.cc

```diff
@@ -62,7 +62,51 @@ except: ValueError
 
 See Also
 --------
+
 - `Tutorial on joins <../../start/quick-start.html#join>`_
+
+Examples
+--------
+
+.. code-block:: python
+
+    df1 = dt.Frame("""date	       X1	X2
+            	01-01-2020	H	10
+            	01-02-2020	H	30
+            	01-03-2020	Y	15
+            	01-04-2020	Y	20""")
+
+    df2 = dt.Frame("""X1	X3
+            	   H	5
+                       Y	10""")
```

Comment:
Could you please reformat with "convert Tabs to Spaces", i think that's what's causing everything to look misaligned.

Reply:
If you don't mind @st-pasha explaining what you mean ? The misalignment is bugging me as well

Reply:
In most IDEs there is a function "convert all tabs to spaces", and also a setting to "always insert tabs as spaces". I don't know which editor you're using, but I think googling this function for your particular editor will tell you where to find it.